src/agreement/mod.rs

```diff
@@ -201,6 +211,7 @@ impl<NodeUid: Clone + Debug + Ord> Agreement<NodeUid> {
                     netinfo,
                     Nonce::new(invocation_id.as_ref(), session_id, proposer_i, 0),
                 ),
+                coin_schedule: CoinSchedule::False,
```

Comment:
I think there was an argument for starting with `True`: In the best case scenario, where everyone gets their proposals delivered in time, everyone would vote `true` in every instance, and it would be nice to terminate in the first round then.

Also, it is probably generally preferred to err on the `true` side and include more elements in the common subset, and therefore more transactions in the epoch.